frontend/src/pages/AdminAddProductPage.tsx

```diff
@@ -1,27 +1,38 @@
 import React, {useState} from "react";
 import axios from "axios";
 
+
 type AdminAddProductPage = {
-    handleClickProduct:()=>void
+    handleClickProduct: () => void
+    unitType: string[]
 }
-export default function AdminAddProductPage(props:Readonly<AdminAddProductPage>) {
+export default function AdminAddProductPage(props: Readonly<AdminAddProductPage>) {
 
     const [name, setName] = useState('');
+    const [unit, setUnit] = useState('');
+    const [amount, setAmount] = useState('');
     const [price, setPrice] = useState('');
     const [error, setError] = useState<string | null>(null);
     const [success, setSuccess] = useState<string | null>(null);
 
+
     const handleSubmit = async (event: React.FormEvent) => {
         event.preventDefault();
         try {
             await axios.post(`/api/product`, {
                 name,
```

Comment:
hier würde ich überlegen den axios-aufruf in eine separate methode zu packen.
deine handleSubmit würde dann nur den submit der form beinhalten und es ist einfacher nachzuvollziehen

Reply:
Meinst du, dass ich eine weitere funktion schreiben soll, die den axios abruf enthält und diese funktion dann in meinen handleSubmit implementieren soll?
Ich brauche ja den Post um ein neues Produkt anzulegen.